src/app/subject/3-d-art-and-design/page.tsx

```diff
@@ -0,0 +1,172 @@
+import { Accordion } from "@/components/ui/accordion";
+import Footer from "@/components/ui/footer";
+import Navbar from "@/components/ui/navbar";
+import { type Subject } from "@/types";
+import SubjectBreadcrumb from "@/components/ui/subject-homepage/subject-breadcrumb";
+import SubjectSidebar from "../../../components/ui/subject-homepage/subject-sidebar";
+import TableOfContents from "../../../components/ui/subject-homepage/table-of-contents";
+import UnitAccordion from "@/components/ui/subject-homepage/unit-accordion";
+
+const Page = ({ params }: { params: { slug: string } }) => {
+  const subject = {
+    title: "AP Calculus AB",
+    units: [
+      {
+        unit: 1,
+        title: "Limits and Continuity",
+        chapters: [
+          {
+            chapter: 1,
```

Comment:
instead of using chapter: number, use index+1 (this is already an array)

Reply:
I decided to leave it there for easier maintanance since it becomes a lot more clear for ppl to see it.